library/Vanilla/Formatting/Quill/BlotGroup.php

```diff
@@ -394,6 +394,10 @@ public function getUnsafeText(): string {
                 $text .= $blot->getContent();
             }
 
+            if ($blot instanceof ExternalBlot) {
+                return '(Embedded Media)';
```

Comment:
This is going to need to be translated. Maybe double check with todd if it's the actual wording we want to use.

Reply:
The Embedded Media is now the "last resort" if there is no embed type specified (ie youtube, image, file, etc.) I didn't bother Todd with it.